core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -2087,6 +2087,17 @@ package object config {
       .doubleConf
       .createOptional
 
+  private[spark] val SCHEDULER_MIN_RESOURCES_TO_SURVIVE_RATIO =
+    ConfigBuilder("spark.scheduler.minResourcesToSurviveRatio")
```

Comment:
I think this config should have the excludeOnFailure in the name if it is applying to that feature, which is implied in description of this.  I also think this feature could be quite confusing to users, should be mentioned in the that documentation.

Reply:
But `excludeOnFailure` is irrelevant, we are applying feature to ExecutorFailureTracker not the HealthTracker

Reply:
Hi @tgravescs, can you take a look at this? Can you explain why this is related to the blacklisting/excludeOnFailure? Did I understand something wrong here?

Reply:
I misread the title, sorry ignore this.